packages/varlet-ui/src/swipe/docs/en-US.md

```diff
@@ -174,11 +180,9 @@ import { Snackbar } from '@varlet/ui'
     <var-swipe-item>
       <img class="swipe-example-image" src="https://varlet.gitee.io/varlet-ui/cat3.jpg">
     </var-swipe-item>
-
     <template #indicator="{ index, length }">
-      <div class="swipe-example-indicators">
-        {{ index + 1 }} / {{ length }}
-      </div>
+      <div class="swipe-example-indicators-left" :class="index === 0 ? 'swipe-example-active-indicator' : 'swipe-example-inactive-indicator'" @click="index !== 0 && swipeRef.prev()"><var-icon name="chevron-left" /></div>
+      <div class="swipe-example-indicators-right" :class="index === length - 1 ? 'swipe-example-active-indicator' : 'swipe-example-inactive-indicator'" @click="index !== length - 1 && swipeRef.next()"><var-icon name="chevron-right" /></div>
```

Comment:
The inline logic is too complicated, it is recommended to extract it into a function.

Reply:
ok, I will delete it.